swagger/public_v1/template.yml

```diff
@@ -435,6 +435,66 @@ components:
             $ref: "#/components/schemas/LocationResource"
         jsonapi:
           $ref: "#/components/schemas/JSONAPI"
+    LocationStatus:
+      type: object
+      description: Wrapper object for LocationStatusAttributes
+      properties:
+        id:
+          type: string
+          example: "22329867"
+        type:
+          type: string
+          example: site_statuses
+        attributes:
+          $ref: "#/components/schemas/LocationStatusAttributes"
+    LocationStatusAttributes:
+      type: object
+      description: Fields relating the state of a location
+      properties:
+        status:
+          type: string
+          enum:
+            - discontinued
+            - running
+            - new_status
+            - suspended
+          example: "running"
+        publish:
+          type: string
+          enum:
+            - published
+            - unpublished
+          example: "published"
+        has_vacancies:
+          type: boolean
+          description: "Are there any vacancies for this course"
+          example: true
+        vac_status:
```

Comment:
Can we call this something a little bit more sensible, like `vacancy_status`?

Reply:
done